data/dialog phrases.txt

```diff
@@ -100,7 +101,8 @@ phrase "generic bounty hunting boarding on visit"
 
 phrase "generic hunted bounty fleet eliminated dialog"
 	word
-		`The <npc> and their fleet have been eliminated. You can claim the bounty payment by returning to <destination>.`
+		`The <npc> and their fleet have been eliminated. Shortly afterwards, the government of <planet> sends you <payment> along with a message of gratitude.`
+
```

Comment:
Instead of changing these dialogs, I think it might make sense to instead create a new dialog with the new text and have that be used by the bounty jobs that have been changed, allowing bounty jobs that require returning to the destination to use the old dialog unchanged.